tasks/src/robocorp/tasks/_commands.py

```diff
@@ -229,16 +228,17 @@ def run(
         max_files=max_log_files,
         max_file_size=max_log_file_size,
     ), setup_log_output_to_port(), context.register_lifecycle_prints():
-        run_status = "PASS"
-        setup_message = ""
-
         run_name = os.path.basename(p)
         if task_name:
             run_name += f" - {task_name}"
 
+        # Status string from `log` module
+        # TODO: Replace with enum
+        run_status: Union[Literal["PASS"], Literal["ERROR"]] = "PASS"
```

Comment:
Question: is there a reason you went with the union of literals here and not just the Status enum directly? Does something break when doing that?

Reply:
... although I guess I could use the task status enum here as well, and then just convert when calling log methods, but not sure if really more readable

Reply:
Ok, agreed.